src/components/smart_objects/src/object_schema_item.cc

```diff
@@ -127,6 +127,7 @@ errors::eType CObjectSchemaItem::validate(
     std::set<std::string>::const_iterator key_it = object_keys.find(key);
     if (object_keys.end() == key_it) {
       if (correct_member && correct_member->mIsMandatory == true &&
```

Comment:
@SNiukalov Can we change it to more short & simple line: 
```cpp
if (correct_member && correct_member->mIsMandatory &&
    !correct_member->mIsDeprecated &&
    !correct_member->mIsRemoved)
```

Reply:
@AByzhynar 
Thanks, fixed in:https://github.com/smartdevicelink/sdl_core/pull/3333/commits/8340fe94900632930a4438cda26dea613e9f6282